### PR TITLE
Fix some cs issues and remove visitor.php from phpcs.xml.dist

### PR DESCRIPTION
**Pull request by IanDelMar**
Avoid Travis result from being polluted by CS errors

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -8,5 +8,7 @@
         <exclude name="Generic.Files.LineLength"/>
         <exclude name="PEAR.Commenting.ClassComment"/>
         <exclude name="PEAR.Commenting.FileComment"/>
+        <exclude name="Squiz.PHP.CommentedOutCode.Found"/>
```
Comment (szepeviktor) [inline phpcs.xml.dist:11]: Please add `// phpcs:disable Squiz.PHP.CommentedOutCode` at the top of that one file

Comment (szepeviktor) [reply phpcs.xml.dist:11]: This would ignore that error in all files.

File: visitor.php [hunk 2]
```diff
@@ -88,7 +89,7 @@ public function format(): array
                     '%s %s%s',
                     $this->tag,
                     $this->type,
-                    ($this->name !== null) ? (' $' . $this->name) : ''
+                    $this->name !== null ? " \$$this->name" : ''
```
Comment (szepeviktor) [inline visitor.php:92]: Please use sprintf or wrap variables in `"{$var}"`

Comment (szepeviktor) [reply visitor.php:92]: If you mix string literals with variable viktor cannot figure out what is going on (👁️ strain)

Comment (IanDelMar) [reply visitor.php:92]: :ok_hand:

**Comment by szepeviktor** [comment]
This multiple-condition array shape handling makes me code!!
Please watch the next PR!

**Pull request merged**